pkg/controller/kccmanager/kccmanager.go

```diff
@@ -185,8 +187,20 @@ func New(ctx context.Context, restConfig *rest.Config, cfg Config) (manager.Mana
 	}
 	rd.DependencyTracker = gcpwatch.NewDependencyTracker(fetcher)
 
+	pollInterval := 10 * time.Minute
+	if interval := os.Getenv("DEPENDENCY_TRACKER_POLL_INTERVAL"); interval != "" {
```

Comment:
If we only plan to support this in testing, maybe we call it `TEST_DEPENDENCY_TRACKER_POLL_INTERVAL`